tests/amqpbasicproperties.phpt

```diff
@@ -0,0 +1,163 @@
+--TEST--
+AMQPBasicProperties
```

Comment:
This is amqp properties that for some reason was mixed with envelope (see it [in java](http://www.rabbitmq.com/releases/rabbitmq-java-client/v3.3.0/rabbitmq-java-client-javadoc-3.3.0/com/rabbitmq/client/Envelope.html)) fields. `basic.return` method holds propertis and no enveloper, so to not confuse userland, `AMQPBasicProperties` introducd and now `AMQPMessage` extends it. Later we'll get rid of `AMQPMessage` or at least make it looks more logical.